src/server_manager/electron_app/release/notarize.js

```diff
@@ -29,9 +29,10 @@ exports.default = async function (context) {
   const appName = context.packager.appInfo.productFilename;
   console.log(`Notarizing ${appName}. This may take a few minutes.`);
   await notarize({
-    appBundleId: 'com.electron.outline-manager',
+    tool: 'notarytool',
     appPath: `${appOutDir}/${appName}.app`,
     appleId: process.env.APPLE_ID,
     appleIdPassword: process.env.APPLE_PASSWORD,
```

Comment:
Can you add a comment saying this should be an app-specific password because of 2FA and point to https://support.apple.com/en-us/HT204397?

Reply:
Sure, I think i'll add an `APPLE_TEAM_ID` env var as well